easypermissions/src/main/java/pub/devrel/easypermissions/AppSettingsDialog.java

```diff
@@ -39,14 +40,16 @@ private AppSettingsDialog(@NonNull final Object activityOrFragment,
         AlertDialog.Builder dialogBuilder = new AlertDialog.Builder(context);
 
         // Set rationale
-        dialogBuilder.setMessage(rationale);
+        dialogBuilder.setMessage(
+                TextUtils.isEmpty(rationale) ? context.getString(R.string.rationale_ask_again) : rationale);
 
         // Set title
-        dialogBuilder.setTitle(title);
+        dialogBuilder.setTitle(
+                TextUtils.isEmpty(title) ? context.getString(R.string.title_settings_dialog) : title);
 
         // Positive button text, or default
         String positiveButtonText = TextUtils.isEmpty(positiveButton) ?
-                context.getString(android.R.string.ok) : positiveButton;
+                context.getString(R.string.settings) : positiveButton;
```

Comment:
@samtstern What do you suggest doing? Should we keep `android.R.string.ok` or add a string for cancel so it isn't translated?